backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -675,6 +675,7 @@ export class AdminResolver {
     { currentPage = 1, pageSize = 5, order = Order.DESC }: Paginated,
   ): Promise<ContributionLinkList> {
     const [links, count] = await DbContributionLink.findAndCount({
+      where: [{ validTo: MoreThan(new Date()) }, { validTo: IsNull() }],
```

Comment:
Do we have `validTo: IsNull()` in the db?
We can not create them in the admin interface as far I have seen.

Reply:
You are right. But the database model allows it. Therefore I put this in